packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -121,6 +122,12 @@ class CupertinoPicker extends StatefulWidget {
   ///
   /// This can be set to null to disable the background painting entirely; this
   /// is mildly more efficient than using [Colors.transparent].
+  ///
+  /// The [backgroundColor] is used to create the gradient of the picker. The
+  /// gradient is added on top of a background of solid [backgroundColor]. Due
```

Comment:
Don't narrate in docs. Just `any alpha value less 255 (fully opaque) will cause the removal of the gradient from rendering of the widget`